index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Landing Page</title>
+<meta charset="UTF-8">
+<link rel="stylesheet" href="style.css">
+</head>
+<body>
+<header id="header">
+    <img id = "header-img" src = "https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.istockphoto.com%2Fphotos%2Flone-wolf&psig=AOvVaw262V3pwgOca0kyLRc0_dWN&ust=1645095402108000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCKCH-4yIhPYCFQAAAAAdAAAAABAD" alt = "logo" class="logo"/>
+    <nav id="nav-bar">
+        <ul>
+          <a
+              class="nav-link" href = "#home">Home</a>
+            <a
+               class="nav-link" href = "#about">About Us</a>
+            <a
+
+                <button class="nav-link" href = "#Contact">Contact Us</a>
+            </a>
+        </ul>
+    </nav>
+
+</header>
+
+<section id="home">
+  <h2>Contact Us</h2>
+  <p>
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+  </p>
+</section>
+  <section id="about" class="class = "flex-container"
```

Comment:
Class is not defined correctly. Also, there shouldn't be any spaces within the attribute